File: q969/death5_1.go
package main
import "fmt"
func main(){
for i:=1;i<51;i++ { if i%15>0 { if i%5>0 { if i%3>0 { fmt.Print(i) } else { fmt.Print("fizz") } } else { fmt.Print("buzz") } } else { fmt.Print("FIZZBUZZ") } }
}

File: tbpgr_colosseum_manager/q969/death5_1.erl
-module(prog).
-export([main/0]).
main()->lists:map(fun(I)->io:fwrite(if
	I rem 15=:=0 -> "FIZZBUZZ";
	I rem 3=:=0 -> "fizz";
	I rem 5=:=0 -> "buzz";
	true -> integer_to_list(I)
end)end,lists:seq(1,50)).